VRDR/DeathRecord_ctors.cs

```diff
@@ -0,0 +1,665 @@
+            return codccBundle;
+        }
+
+        /// <summary>Helper method to return the subset of this record that makes up a DemographicCodedContent bundle.</summary>
+        /// <returns>a new FHIR Bundle</returns>
+        public Bundle GetDemographicCodedContentBundle()
+        {
+            Bundle dccBundle = new Bundle();
+            dccBundle.Id = Guid.NewGuid().ToString();
+            dccBundle.Type = Bundle.BundleType.Collection;
+            dccBundle.Meta = new Meta();
+            string[] profile = { ProfileURL.DemographicCodedContentBundle };
+            dccBundle.Meta.Profile = profile;
+            dccBundle.Timestamp = DateTime.Now;
+            // Make sure to include the base identifiers, including certificate number and auxiliary state IDs
+            dccBundle.Identifier = Bundle.Identifier;
+            AddResourceToBundleIfPresent(CodedRaceAndEthnicityObs, dccBundle);
+            AddResourceToBundleIfPresent(InputRaceAndEthnicityObs, dccBundle);
+            return dccBundle;
+        }
+        /// <summary>Helper method to return the subset of this record that makes up a Mortality Roster bundle.</summary>
+        /// <returns>a new FHIR Bundle</returns>
+        public Bundle GetMortalityRosterBundle(Boolean alias)
+        {
+            Bundle mortRosterBundle = new Bundle();
+            mortRosterBundle.Id = Guid.NewGuid().ToString();
+            mortRosterBundle.Type = Bundle.BundleType.Collection;
+            mortRosterBundle.Meta = new Meta();
+            string[] profile = { ProfileURL.MortalityRosterBundle };
+            mortRosterBundle.Meta.Profile = profile;
+            mortRosterBundle.Timestamp = DateTime.Now;
+            mortRosterBundle.Identifier = Bundle.Identifier; // includes the certificate number, and aux state IDs
+            AddResourceToBundleIfPresent(Decedent, mortRosterBundle);
+            AddResourceToBundleIfPresent(DeathLocationLoc, mortRosterBundle);
+            AddResourceToBundleIfPresent(DeathDateObs, mortRosterBundle);
+            AddResourceToBundleIfPresent(Father, mortRosterBundle);
+            AddResourceToBundleIfPresent(Mother, mortRosterBundle);
+
+            // Stick Replace and Alias into bundle header as extensions
+            // Copy replace from Composition header
+            // Use value of alias from argument
+            if (!String.IsNullOrWhiteSpace(ReplaceStatusHelper))
+            {
+                    Extension replaceExt = new Extension(ExtensionURL.ReplaceStatus , DictToCodeableConcept(ReplaceStatus) );
+                    mortRosterBundle.Meta.Extension.Add(replaceExt);
+            }
+            Extension aliasExt = new Extension(ExtensionURL.AliasStatus, new FhirBoolean(alias));
+            mortRosterBundle.Meta.Extension.Add(aliasExt);
+            return mortRosterBundle;
+        }
+
+
+/////////////////////////////////////////////////////////////////////////////////
```

Comment:
The indenting on this line seems to have gotten munged.

Maybe run a formatter on all the files, since this is a time when the noise in a diff won't impact anything?